src/content/docs/ru/guides/imports.mdx

```diff
@@ -0,0 +1,290 @@
+
+`Astro.glob()` принимает только один параметр: относительный [шаблон glob](/ru/guides/imports/#шаблоны-glob), соответствующий локальным файлам, которые вы хотите импортировать. Это асинхронная функция, которая возвращает массив экспортов каждого соответствующего файла.
+
+```astro title="src/components/my-component.astro"
+---
+// импортирует все файлы, оканчивающиеся на `.md` в `./src/pages/post/`
+const posts = await Astro.glob('../pages/post/*.md'); 
+---
+<!-- Отображает <article> для первых 5 блог-постов -->
+<div>
+{posts.slice(0, 4).map((post) => (
+  <article>
+    <h2>{post.frontmatter.title}</h2>
+    <p>{post.frontmatter.description}</p>
+    <a href={post.url}>Read more</a>
+  </article>
+))}
+</div>
+```
+
+Astro компоненты, импортированные с помощью `Astro.glob`, имеют тип [`AstroInstance`](/ru/reference/api-reference/#astro-files). Вы можете отобразить каждый экземпляр компонента, используя его свойство `default`:
+
+```astro title="src/pages/component-library.astro" {8}
+---
+// импортирует все файлы, оканчивающиеся на `.astro` в `./src/components/`
+const components = await Astro.glob('../components/*.astro');
+---
+<!-- Отображает все наши компоненты -->
+{components.map((component) => (
+  <div>
+    <component.default size={24} />
+  </div>
+))}
+```
+
+
+### Шаблоны Glob
+
+Шаблон glob - это путь к файлу, который поддерживает специальные символы подстановки. Это используется для ссылки на несколько файлов в вашем проекте одновременно.
+
+Например, шаблон glob `./pages/**/*.{md,mdx}` начинается в подкаталоге pages, просматривает все его подкаталоги (`/**`) и соответствует любому имени файла (`/*`), которое заканчивается на `.md` или `.mdx` (`.{md,mdx}`).
+
+#### Шаблоны Glob в Astro
+
+Для использования с `Astro.glob()`, шаблон glob должен быть строковым литералом и не может содержать переменных. См. [руководство по устранению неполадок](/ru/guides/troubleshooting/#astroglob---no-matches-found) для обходного пути.
+
+Кроме того, шаблоны glob должны начинаться с одного из следующих:
+- `./` (чтобы начать в текущем каталоге)
+- `../` (чтобы начать в родительском каталоге)
+- `/` (чтобы начать с корня проекта)
+ 
+
+[Узнайте больше о синтаксисе шаблонов glob](https://github.com/mrmlnc/fast-glob#pattern-syntax).
+
+#### `Astro.glob()` vs `getCollection()`
+
+[Коллекции контента](/ru/guides/content-collections/) предоставляют API [`getCollection()`](/ru/reference/modules/astro-content/#getcollection) для загрузки нескольких файлов вместо `Astro.glob()`. Если ваши файлы контента (например, Mark загружает и инициализирует запрошенный файл коллекции в каталоге `src/content/`), используйте `getCollection()`, чтобы [запросить коллекцию](/ru/guides/content-collections/#querying-collections) и вернуть записи контента.
+
+<ReadMore>Узнайте больше о [подключении коллекций](/ru/guides/content-collections/#запрос-к-коллекциям) и возврате записей контента.</ReadMore>
```

Comment:
Looks like I made a mistake!

This part doesn't seem to exist in the original documentation.